abcix/adapter/adapter.go

```diff
@@ -32,59 +35,66 @@ func (app *adaptedApp) OriginalApp() abci.Application {
 	return app.abciApp
 }
 
-func (app *adaptedApp) Info(req abcix.RequestInfo) (resp abcix.ResponseInfo) {
-	abciReq := abci.RequestInfo{}
-	if err := copier.Copy(&abciReq, &req); err != nil {
+var typeRegistry = map[string]reflect.Type{
+	"abci.RequestInfo":               reflect.TypeOf(abci.RequestInfo{}),
```

Comment:
nit: use const

```
const (
  info = iota
  query
  checkTx
  initChain
...
)
```

Reply:
`reflect.Type` is not a valid constant type, so we might not be able to use constant here.

Reply:
I mean the map key